src/app/workflow/info-tab/info-tab.service.ts

```diff
@@ -187,11 +187,12 @@ export class InfoTabService {
     );
   }
 
-  getTRSId(workflow: Workflow | undefined, entryType: EntryType): string {
-    if (!workflow) {
+  getTRSId(workflow: Workflow | undefined): string {
+    if (workflow) {
+      return workflow.entryTypeMetadata.trsPrefix + workflow.full_workflow_path;
```

Comment:
No need to change, but FYI, there is a `trsId` field now:  https://github.com/dockstore/dockstore/blob/c8459501ac20132294376702c6c8a81338d3c9bf/dockstore-webservice/src/main/resources/openapi3/openapi.yaml#L12529-L12530

Reply:
the `trsId` should be available for `Workflow` https://github.com/dockstore/dockstore/blob/c8459501ac20132294376702c6c8a81338d3c9bf/dockstore-webservice/src/main/java/io/dockstore/webservice/core/Entry.java#L969-L971

Reply:
Forgot about that, changed.  Thanks Kathy!

Reply:
Woops, something is wrong.  Being that I'm leaving soon and we're trying to deploy again, reverting to original solution...